.circleci/config.yml

```
@@ -90,16 +93,20 @@ commands:
                 -DFL_BUILD_APP_IMGCLASS=<< parameters.fl_build_app_imgclass >> \
                 -DFL_BUILD_APP_OBJDET=<< parameters.fl_build_app_objdet >> \
                 -DFL_BUILD_APP_LM=<< parameters.fl_build_app_lm >> \
-                -DBUILD_SHARED_LIBS=<< parameters.build_shared_libs >> \
+                -DFL_BUILD_ALL_PKGS=<< parameters.fl_build_all_pkgs >> \
+                -DFL_BUILD_SHARED_LIBS=<< parameters.build_shared_libs >> \
```

`BUILD_SHARED_LIBS` is a standard CMake option -- I don't think we want this change
```suggestion
                -DBUILD_SHARED_LIBS=<< parameters.build_shared_libs >> \
```

---

Ah yes. Good catch. Unintentional.